nebu/models/document.py

```diff
@@ -107,6 +107,7 @@ def _find_resources(loc):
         :rtype: [:class:`cnxepub.models.Resource`]
 
         """
+        # FIXME: This is _technically_ used, however, it serves no purpose
```

Comment:
Some more details: I think this picks up resources and will sym link them post-assembly. As far as I know, however, we only use the assembled xhtml files.

Reply:
Yep, :+1: for removing